install_config/topics/configuring_for_gce_registry-proc.adoc

```diff
@@ -0,0 +1,289 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+= Configuring the {product-title} registry for GCP
+
+Google Cloud Platform (GCP) provides object cloud storage that {product-title}
+can use to store container images using the {product-title} container registry.
+
+For more information, see link:https://cloud.google.com/storage/docs/[`Cloud Storage` in the GCP documentation].
+
+.Prerequisites
+
+You must create the bucket to host the registry images before the installation.
+The following commands create a regional bucket using the configured service
+account:
```

Comment:
Would it be more accurate to say "You must create the storage bucket...? And then link to these docs:

https://cloud.google.com/storage/docs/creating-buckets

Reply:
If we don't want to show any specific GCP commands, sure, remove the gsutil stuff

Reply:
Hmm i linked to those docs instead of taking away any info. If it's specific to OCP, then perhaps it's best to keep it.